integrate it with pytorch

fenypatel99 opened the pull request:
Differential Revision: D57172228

aaronenyeshi commented on torch/csrc/profiler/kineto_shim.cpp:
```diff
@@ -362,6 +363,18 @@ c10::DeviceType deviceTypeFromActivity(libkineto::ActivityType activity_type) {
       }();
       return device_type;
     }
+    case libkineto::ActivityType::MTIA_WORKLOADD: {
```
You can combine this option to line 354, instead of duplicating this block

fenypatel99 committed:
integrate workloadd events with pytorch (#126508)

Summary:
integrate workloadd events with pytorch

Test Plan: CIs

Differential Revision: D57172228
